frontend/packages/core/src/Input/text-field.tsx

```diff
@@ -92,6 +100,40 @@ const StyledTextField = styled(BaseTextField)({
   },
 });
 
+// popper containing the search result options
+const Popper = styled(MuiPopper)({
+  ".MuiAutocomplete-paper": {
+    boxShadow: "0px 5px 15px rgba(53, 72, 212, 0.2)",
+  },
+  ".MuiAutocomplete-option": {
+    height: "48px",
+    padding: "0px",
+  },
+  ".MuiAutocomplete-option[data-focus='true']": {
```

Comment:
I'd need to play around with this component but I believe you can use the focus state similar to [here](https://github.com/lyft/clutch/blob/main/frontend/packages/core/src/Input/toggle.tsx#L11)

Reply:
I just used all of the styling that was done for the search component https://github.com/lyft/clutch/blob/main/frontend/packages/core/src/AppLayout/search.tsx#L102-L118, I didn't change much with the css besides the border of the result box that was casting a weird shadow.